xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAEFilter.cpp

```diff
@@ -277,6 +280,8 @@ int CActiveAEFilter::ProcessFilter(uint8_t **dst_buffer, int dst_samples, uint8_
       return -1;
     }
 
+    m_SamplesOut = m_pOutFrame->pts;
```

Comment:
oh, this should be outFrame, not m_pOutFrame. if format conversion is required outFrame is set to m_pConvertFrame. that is the reason why the issue showed on Pi and not on Ubuntu